GW2EIEvtcParser/EncounterLogic/Fractals/FractalLogic.cs

```diff
@@ -146,26 +147,41 @@ protected static long GetFightOffsetByFirstInvulFilter(FightData fightData, Agen
         internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedEvtcLog log, CombatReplay replay)
         {
             base.ComputePlayerCombatReplayActors(p, log, replay);
-            if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.ToxicSicknessPuke1, out IReadOnlyList<EffectEvent> sicknessEffects))
+
+            // Toxic Sickness - Old Indicator
```

Comment:
var toxicSicknessGUIDs = new List<String>() { EffectGUIDs.ToxicSicknessOldIndicator, EffectGUIDs.ToxicSicknessNewIndicator};
then iterate over that list, the content of the 2 ifs is identical.